sources/CascadiaCode-Bold.ufo/fontinfo.plist

```diff
@@ -32,7 +32,7 @@
         <key>x</key>
         <integer>886</integer>
         <key>y</key>
-        <integer>-20</integer>
+        <integer>-1040</integer>
```

Comment:
these guideline changes aren't anything to worry about, right?

Reply:
Nah. That's just a guideline. Won't affect anything.